apps/mobile/src/navigation/UserSettingsTabsNavigator.js

```diff
@@ -85,42 +85,42 @@ export default function UserSettingsTabsNavigator ({ navigation, route }) {
         name='Edit Profile'
         component={UserSettingsWebView}
         initialParams={{
-          path: '/settings'
+          path: '/my/edit-profile'
```

Comment:
I was trying to navigate to these webviews with the existing structure and haven't figured it out. But do they even need a separate settings tab/navigator now? Could these screens just be moved over to the home navigator?